fhir-model/src/test/java/com/ibm/fhir/model/test/TestUtil.java

```diff
@@ -240,4 +242,39 @@ public static JsonObject readJsonObject(String fileName) throws Exception {
             
         }
     }
+
+    /**
+     * If the {@code resourceToFind} is contained in the list of resources this method returns the resource.
+     * Otherwise it returns null.
+     * @param resources
+     */
+    public static boolean findResourceInResponse(Resource resourceToFind, List<? extends Resource> resources) {
```

Comment:
update javadoc to indicate which checks are performed, or update the name to something like `isResourceInResponseExactlyOnce` as we discussed

Reply:
maybe just go with `isResourceInResponse` and update the javadoc to indicate that it will fail the test if we either:
A. find the same resource twice; or
B. find different versions of the same resource

Reply:
done